deploy.sh

```diff
@@ -62,3 +68,9 @@ git commit -m "Deploy to GitHub Pages: ${SHA}"
 
 # Now that we're all set up, we can push.
 git push $SSH_REPO $TARGET_BRANCH
+
+function TRpush {
+  curl "https://labs.w3.org/echidna/api/request" --data "url=https://heycam.github.io/webidl/echidna.html" --data "decision=https://lists.w3.org/Archives/Public/public-webapps/2014JulSep/0627.html" --data "token=$TOKEN"'
```

Comment:
Oh, if a hook-based solution avoids this, and @heycam is reachable, I'd rather we go that route.